tests/mutations/test_mutations.py

```diff
@@ -95,3 +97,245 @@ def test_delete_with_filters(mutation, fruits):
     assert list(models.Fruit.objects.values("id", "name")) == [
         {"id": 3, "name": "banana"},
     ]
+
+
+@pytest.mark.skipif(
+    not settings.GEOS_IMPORTED,
+    reason="Test requires GEOS to be imported and properly configured",
+)
+@pytest.mark.django_db(transaction=True)
+def test_create_geo(mutation):
+    from tests.models import GeosFieldsModel
+
+    # Test for point
+    point = [0.0, 1.0]
+    result = mutation(
+        "{ geofield: createGeoField(data: { point: " + str(point) + " }) { id } }"
+    )
+    assert not result.errors
+    assert (
+        deep_tuple_to_list(
+            GeosFieldsModel.objects.get(id=result.data["geofield"]["id"]).point.tuple
+        )
```

Comment:
Why transform the tuples into lists instead of comparing them as tuples? (the same applies to the whole test code)

Reply:
Cause from what I tested, tuples will give unexpected results when comparing larger tuples.
E.g. 
`( ( 0, 1 ), ( 1, 1) )` will be deemed different compared to 
`( ( 0, 1 ), ( 1, 1), )`

Reply:
@bellini666 turns out you're right! For some reason tests failed previously and I have no idea why.
I'm now using deep_tuple_to_list only when generating queries cause the mutation input format requires it hehe.

4531b6a0ba1a2431ebcad17909a575115a678796